webapp/graphite/render/functions.py

```diff
@@ -4049,6 +4050,39 @@ def randomWalkFunction(requestContext, name, step=60):
             int(epoch(requestContext["endTime"])),
             step, values)]
 
+def seriesByTag(requestContext, *tagExpressions):
```

Comment:
This is designed to support query syntax like `seriesByTag('tag1=value1', 'tag2=value2')`